tools/ci/build-and-check-r-package.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/sh
+
+cd apis/R
+
+echo "::group::Build TileDB R package"
+## build package
+R CMD build --no-build-vignettes --no-manual .
+echo "::endgroup::"
+
+echo "::group::Check TileDB R package"
+## set flag to tolerate missing suggested packages (as e.g. vignette-building tools)
+export _R_CHECK_FORCE_SUGGESTS_=FALSE
+## check package
+R CMD check --as-cran --no-manual --ignore-vignettes tiledb_*.tar.gz
```

Comment:
That is a likely bug.  The package you copied this from is called `tiledb_X.Y.Z.tar.gz` so the file-globbing is correct.  Your package here appears to be `tiledbsoma` per the `Package: tiledbsoma` in `DESCRIPTION` so you need to take out the underscore there, or inject 'soma' in front of it.

Reply:
https://github.com/single-cell-data/TileDB-SOMA/pull/296

thank you! :)